src/lib/features/feature-toggle/feature-toggle-service.ts

```diff
@@ -2402,6 +2402,10 @@ class FeatureToggleService {
             );
         }
     }
+
+    async setFeatureCreatedByUserIdFromEvents(): Promise<void> {
+        await this.featureToggleStore.setCreatedByUserId(500);
```

Comment:
What's a reasonable ratio of features per run?

Reply:
I'd say 50... we have an average of ~50 toggles per customer. But this batch size is not about the number of feature toggles, it's more about the rows returned from the query

Reply:
100 every 15 minutes after looking at real data might be a good thing. This should most likely finish the job in 10 days even for large tables

Reply:
Has been set to 100 every 15mins as discussed!